management/src/main/java/io/micronaut/management/health/aggregator/RxJavaHealthAggregator.java

```diff
@@ -23,10 +23,10 @@
 import io.micronaut.management.health.indicator.HealthIndicator;
```

Comment:
Class needs renaming since it doesn't use RxJava anymore

Reply:
Renamed to `DefaultHealthAggregator` https://github.com/micronaut-projects/micronaut-core/pull/5583/commits/8a765fcd35643f0ac6aa9b3dd995ca297606594b